iceoryx_posh/include/iceoryx_posh/internal/roudi/introspection/port_introspection.hpp

```diff
@@ -330,11 +333,8 @@ class PortIntrospection
   private:
     PortData m_portData;
 
-    std::atomic<bool> m_runThread;
-    std::thread m_thread;
-
-    unsigned int m_sendIntervalCount{10};
-    const std::chrono::milliseconds m_sendIntervalSleep{100};
+    units::Duration m_sendInterval{units::Duration::seconds<long double>(1.0)};
```

Comment:
That's currently a limitation of `units::Duration`. If I'm not completely a moron, that's the only way to create a duration.

Reply:
Well, apparently I am a moron. One could also use `unsigned long long int` instead of `long double`.

In my shame I replace the `long double` internal representation to a fixed size in #501 to repent my stupidity 😁